whois-api/src/test/java/net/ripe/db/whois/api/rest/SyncUpdatesServiceTest.java

```diff
@@ -31,6 +31,7 @@
 import java.net.HttpURLConnection;
 import java.util.Collections;
 
+import static javax.ws.rs.core.Response.Status.UNAUTHORIZED;
```

Comment:
Use `HttpStatus` not `javax.ws.rs.core.Response.Status` for consistency

Reply:
Authclient is part of whois-client module and we do not have org.eclipse.jetty. In this module we use javax.ws.rs.core.Response.Status.INTERNAL_SERVER_ERROR in other classes as well

Reply:
OK, if whois-client is already different then use `Response.Status` (for now, we should refactor this).